nvflare/private/fed/server/job_cmds.py

```diff
@@ -162,7 +169,13 @@ def get_spec(self):
                     handler_func=self.pull_file,
                     authz_func=self.authorize_job_file,
                     client_cmd=ftd.PULL_BINARY_FQN,
-                    visible=False,
+                ),
+                CommandSpec(
+                    name=AdminCommandNames.DOWNLOAD_JOB_COMPONENTS,
+                    description="download additional components for a specified job",
+                    usage=f"{AdminCommandNames.DOWNLOAD_JOB_COMPONENTS} job_id",
+                    handler_func=self.download_job_components,
+                    client_cmd=ftd.PULL_FOLDER_FQN,
```

Comment:
Instead of a new command, what if we enhance the current "download_job" command to also download all the "components" that belongs to the job?

Reply:
No, this is because download_job_components may have different permission than download_job.